spec/support/dummy_app/Gemfile

```diff
@@ -12,3 +12,4 @@ gem 'unicorn'
 gem 'pg', '~> 1.2'
 gem 'byebug'
 gem 'rails_failover', path: '../../../../rails_failover'
+gem 'psych', '~> 3.0'
```

Comment:
Ruby 3.1 brings Psych 4.0 which changed `YAML.load` to use `.safe_load` so that breaks `database.yml` which contains aliases.